### PR TITLE
file structure

### PR DESCRIPTION
**Pull request by modrayyes**
Front-end file structure (npx create-react-app)

### CODE REVIEW

**Review by NadiaJalalTamimi — DISMISSED** [review]

**Review by ahmadfarid-059 — DISMISSED** [review]

**Comment by Mu7ammadAbed** [comment]
You need to create 2 folders, one for your server and another one for client which should include the react files

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (Mu7ammadAbed) [inline public/index.html:9]: You should edit this

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (Mu7ammadAbed) [inline public/robots.txt:1]: There's no need for this file

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (Mu7ammadAbed) [inline src/App.css:1]: You are not using any of these styles o you should remove them

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import './index.css';
+import App from './App';
+
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
```
Comment (Mu7ammadAbed) [inline src/index.js:12]: You need to add a new line here and I think you should import `BrowserRouter` and wrap your app with it here

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]
You should install eslint and set its configurations

**Review by ahmadfarid-059 — APPROVED** [review]

**Review by ahmadfarid-059 — DISMISSED** [review]

**Review by asmaaamin — DISMISSED** [review]

**Review by ahmadfarid-059 — APPROVED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

**Review by asmaaamin — APPROVED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Pull request merged**